test/config/arrow-function-parameters/main.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @minItems 1
+ */
+export type MyNonEmptyArray<T> = T[];
+/**
+ * @title String field title
+ * @minLength 10
+ * @format date-time
+ * @pattern /^\d+$/
+ */
+export type StringValue = string;
+
+/**
+ * @title String field title
+ * @minLength 10
+ * @format date-time
+ * @pattern /^\d+$/
+ */
+export type NumberValue = string;
+
+export const myFunction = (
```

Comment:
Let's simplify the function to test the essential only.

Reply:
Simplified it in 7afe527